### DISCUSSION TITLE
Can anyone add a package into software sources

### DISCUSSION THREAD

**Discussion by gan-zr** [termux/termux-app#4231] [Ideas]
Acutally ,I want to compile [TinyTex](https://github.com/rstudio/tinytex) for my Termux , but it shows it lack a package named "Kpathse". Additionally ,I found that in some distribution like Debian. Why Termux can't possess it? Sorry for my poor English.

**Comment by TomJo2000** [comment]
We have a [package request template](https://github.com/termux/termux-packages/issues/new?assignees=&labels=package+request&projects=&template=02-package-request.yml&title=%5BPackage%5D%3A+) over in termux/termux-packages.
You can use that to make a package request.

**Comment by Biswa96** [comment]
Do this repository support R language? tinytex is written with that language.

**Reply by TomJo2000** [reply]
Ah I see the problem.
TinyTex is written in R, we don't currently have an interpreter for R.
![image](https://github.com/user-attachments/assets/1d721303-0b3b-404f-9225-2472f6e768fc)
